### ISSUE TITLE
Naming conflict with the new Fable.Browser.XXX packages

### ISSUE THREAD

**Issue by alfonsogarciacaro** [fable-elmish/browser#21]
When discussing the naming for the new Fable.Browser packages [it was decided](https://github.com/fable-compiler/fable-browser/issues/1#issuecomment-457147602) to just use the `Browser` namespace so users only had to `open Browser`, however this is causing conflicts whenever you `open Elmish` and then `open Browser` if this package is being referenced. The F# compiler thinks you're trying to `open Elmish.Browser` through a partially qualified namespace and gives you an error. 

![image](https://user-images.githubusercontent.com/8275461/56279333-2d59c000-6108-11e9-91ee-0b8eeac5740b.png)
 
I've made some tests and it seems we can fix this without creating breaking changes (Fable.Browser.XXX packages have already been released in stable version) by turning `Elmish.Browser` into a module.

```fsharp
module Elmish.Browser

module Option = ...
module UrlParser = ...
module Navigation = ...
```

The only problem is this would force us to use a single file for the package because you cannot split a module across multiple files in F#. What do you think, @et1975? I know it's not ideal but it would improve user experience without asking for more changes. If you agree I can send a PR.

cc @forki @MangelMaxime

**Comment by et1975** [comment]
I wasn't particularly fond of Browser namespace here anyway, I think we can safely get rid of it, see the commit I just pushed.

**Comment by et1975** [comment]
Published as 3.0.0-beta-4

**Comment by alfonsogarciacaro** [comment]
Cool, thanks, I'll update the blog post.
